filament/backend/include/private/backend/DriverAPI.inc

```diff
@@ -427,6 +427,12 @@ DECL_DRIVER_API_N(makeCurrent,
 DECL_DRIVER_API_N(commit,
         backend::SwapChainHandle, sch)
 
+/*
+ * Offscreen
+ */
+
+DECL_DRIVER_API_0(makeCurrentOffscreen)
```

Comment:
Sadly, the name can't be `makeCurrent`, because then Filament can't decide, for which overload wrap the supplied arguments.

Reply:
Could the new function on `OpenGLDriver` follow the naming in that case?

Reply:
fixed in https://github.com/shapr3d/filament/pull/80/commits/1fe120e9b297173062f17650953fe021879874f1